src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -61,9 +65,16 @@ export const getEvent = (
         return getEventHelmApps(ad, nodeName, nodeType, isResourceBrowserView, selectedResource)
     }
     const cn = ad.resourceTree.nodes.filter((node) => node.name === nodeName && node.kind.toLowerCase() === nodeType)[0]
-    return get(
-        `api/v1/applications/${ad.appName}-${ad.environmentName}/events?resourceNamespace=${ad.namespace}&resourceUID=${cn.uid}&resourceName=${cn.name}`,
-    )
+    selectedResource = {
```

Comment:
This `selectedResource` being repeated in line 24 as well. Can we try to optimize it

Reply:
Optimised 👍